### ISSUE TITLE
ViewMode=2 year Muliselected years issue in IE8

### ISSUE THREAD

**Issue by jamiebrohan** [uxsolutions/bootstrap-datepicker#1028]
Set ViewMode=2. Multiselect defaulted to false.

Open link below in IE8
http://eternicode.github.io/bootstrap-datepicker/?markup=component&format=&weekStart=&startDate=&endDate=&startView=0&minViewMode=0&todayBtn=false&language=en&orientation=auto&multidate=&multidateSeparator=&autoclose=on&keyboardNavigation=on&forceParse=on#sandbox

Use calendar button to select a date.
In textbox update the year to a year within the same decade.
Newly selected year and old year are both highlighted.

![image](https://cloud.githubusercontent.com/assets/4640314/3839227/365502a6-1e0e-11e4-85e8-a53d8cb3e629.png)

I got round issue with hack of changing the fill function from

```
        var years = $.map(this.dates, function(d){
                return d.getUTCFullYear();
            }),
            classes;
```

to

```
        var years = $.map(this.dates, function(d){
                return d.getUTCFullYear();
            }),
            classes;
        if (years.length > 1)
            years = years.slice(years.length - 1);
```

Basically slice off all but the last year.
